packages/expo-apple-authentication/src/AppleAuthentication.types.ts

```diff
@@ -0,0 +1,250 @@
+import { StyleProp, ViewStyle } from 'react-native';
+
+export type AppleAuthenticationButtonProps = {
+  onPress: () => void;
+  buttonType: AppleAuthenticationButtonType;
+  buttonStyle: AppleAuthenticationButtonStyle;
+  cornerRadius?: number;
+  style?: StyleProp<ViewStyle>;
+};
+
+/**
+ * The options you can supply when making a call to
```

Comment:
@esamelson In your VS Code settings.json maybe try setting 
```
	"editor.rulers": [
		100
	],
```

I'm not sure if that is the correct setting, but my rewrap plugin wraps at 100 columns, the same as the Prettier config. (We use 100 col instead of 80 because TS has generics, which add to line length. h/t Google)